src/components/Select.tsx

```diff
@@ -14,7 +14,8 @@ const StyledGrommetSelect = styled(GrommetSelect)<InternalSelectProps<any>>`
 	font-family: ${props => props.theme.font};
 	font-size: ${props => px(props.theme.fontSizes[1])};
 	font-weight: normal;
-	height: ${props => (props.emphasized ? '46px' : '36px')};
+	height: ${props =>
+		px((props.emphasized ? props.theme.space[5] : props.theme.space[4]) - 2)};
```

Comment:
Why is the select component 2px smaller than the input component? Surely they should be the same height?

Reply:
@LucianBuzzo that is so they are actually the same size. The problem is the select component's button has 1px borders that make the actual button `40px`. if you change the height of the button to `38px`, then the alignment with the drop is messed up. There isn't a clean way to set the height otherwise. I should probably add a comment though.

Reply:
Nevermind, found a better way to do it.

Reply:
We should be using `box-sizing: border-box`, which should constrain the border within the defined height - is this not the case?

Reply:
It is using `box-sizing: border-box`, but instead of setting the height on the input, it should have been the button.